src/driver/mod.rs

```diff
@@ -144,40 +144,75 @@ impl Drop for Driver {
         }
 
         // pre-determine what to cancel
-        let mut cancellable_ops = Vec::new();
-        for (id, cycle) in self.ops.lifecycle.iter() {
-            // don't cancel completed items
-            if !matches!(cycle, Lifecycle::Completed(_)) {
-                cancellable_ops.push(id);
+        // All Cancallable Ops are marked ignored
```

Comment:
This particular comment is in the wrong  place now. This line could read something like
// Leaving Completed as Completed, CompletionList with final already there also made Completed, turning the rest into Ignored.